open-ce/env_config.py

```diff
@@ -17,18 +18,46 @@
           " for a list of requirements.")
     sys.exit(1)
 
+class Key: #pylint: disable=no-init,too-few-public-methods
+    '''Enum for Env Config Keys'''
+    IMPORTED_ENVS = 0
```

Comment:
Instead of using integers here, I think you can put the actual string. That would make using this much simpler because you wouldn't have to index into ENV_CONFIG_KEYS.

Reply:
That would simplify the usage, but you could never do something like iterate over all the values of the enum.

However. It looks like python 3.4 added some type of enum. Since we're only in Python 3 land now I'll look into this:
https://docs.python.org/3/library/enum.html

Reply:
I changed to the Py3 way of doing enums. Let me know what you think.